dmoj/checkers/bridged.py

```diff
@@ -7,45 +7,28 @@
 from dmoj.utils.aux_files import compile_with_auxiliary_files, mktemp
 from dmoj.utils.unicode import utf8text
 
-checker_defaults = {
-    'time_limit': env['generator_time_limit'],
-    'memory_limit': env['generator_memory_limit'],
-    'compiler_time_limit': env['generator_compiler_limit'],
-    'feedback': True,
-}
-
 executor = None
 
 
-def get_executor(checker_kwargs, problem_id):
+def get_executor(files, lang, compiler_time_limit, problem_id):
     global executor
 
     if executor is None:
-        if 'files' not in checker_kwargs:
-            raise InternalError('no checker file(s) specified!')
-        if 'lang' not in checker_kwargs:
-            raise InternalError('language not specified for checker!')
-
-        filenames = list(map(lambda x: os.path.join(get_problem_root(problem_id), x), checker_kwargs['files']))
-        lang = checker_kwargs['lang']
-        executor = compile_with_auxiliary_files(filenames, lang, checker_kwargs['compiler_time_limit'])
+        filenames = list(map(lambda x: os.path.join(get_problem_root(problem_id), x), files))
```

Comment:
As a general rule, can we stop with the `list(map(...))` nonsense and just use comprehensions?